app/script/view_model/content/TitleBarViewModel.js

```diff
@@ -80,7 +80,9 @@ z.viewModel.content.TitleBarViewModel = class TitleBarViewModel {
     });
 
     this.supportsVideoCall = ko.pureComputed(() => {
-      return this.showCallControls() ? this.conversationEntity().supportsVideoCall(true) : false;
+      return this.showCallControls()
+        ? this.userRepository.self().inTeam() && this.conversationEntity().supportsVideoCall(true)
```

Comment:
You can directly user `this.userRepository.isTeam()` instead of relying on the self user property.